examples/adxl34x_tap_detection_test.py

```diff
@@ -0,0 +1,14 @@
+import time
+import board
+import busio
+import adafruit_adxl34x
+
+i2c = busio.I2C(board.SCL, board.SDA)
+
+accelerometer = adafruit_adxl34x.ADXL345(i2c)
+accelerometer.enable_tap_detection(1, 10)
```

Comment:
Overall I like how this reads. What do you think about making these first two kwarg-only? That way this would be: 
`accelerometer.enable_tap_detection(tap=1, threshold=10)`

`tap` may read better as `tap_count` as well.

Reply:
Agreed on both counts. I'm actually liking the idea of having all the params for the new methods be kwarg-only as I think it would make the code read better.

Reply:
@siddacious Yeah, I like that too. Its particularly handy for methods with multiple params of the same type.

Please mention me when you've changed it. I think we can merge even if @microbuilder hasn't finished looking at it. He can always follow up after if needed.

Reply:
I did a quick HW test and this seems fine to me, though the API decisions are definitely your call!